runtime/src/lib.rs

```diff
@@ -489,6 +497,121 @@ impl pallet_multisig::Config for Runtime {
     type WeightInfo = pallet_multisig::weights::SubstrateWeight<Runtime>;
 }
 
+// Proxy Pallet config
+parameter_types! {
+    // One storage item; key size sizeof(AccountId) = 32, value sizeof(Balance) = 8; 40 total
+    pub const ProxyDepositBase: Balance = deposit(1, 40);
+    // Adding 32 bytes + sizeof(ProxyType) = 32 + 1
+    pub const ProxyDepositFactor: Balance = deposit(0, 33);
+    pub const MaxProxies: u32 = 20; // max num proxies per acct
+    pub const MaxPending: u32 = 15 * 5; // max blocks pending ~15min
+    // 16 bytes
+    pub const AnnouncementDepositBase: Balance =  deposit(1, 16);
+    // 68 bytes per announcement
+    pub const AnnouncementDepositFactor: Balance = deposit(0, 68);
+}
+
+#[derive(
+    Copy,
+    Clone,
+    Eq,
+    PartialEq,
+    Ord,
+    PartialOrd,
+    Encode,
+    Decode,
+    RuntimeDebug,
+    MaxEncodedLen,
+    TypeInfo,
+)]
+pub enum ProxyType {
+    Any,
+    Owner, // Subnet owner Calls
+    NonCritical,
+    NonTransfer,
+    Senate,
+    NonFungibile, // Nothing involving moving TAO
+    Triumvirate,
+    Governance, // Both above governance
+    Staking,
+    Registration,
+}
+impl Default for ProxyType {
+    fn default() -> Self {
+        Self::Any
+    }
+} // allow all Calls; required to be most permissive
+impl InstanceFilter<RuntimeCall> for ProxyType {
+    fn filter(&self, c: &RuntimeCall) -> bool {
+        match self {
+            ProxyType::Any => true,
+            ProxyType::NonTransfer => !matches!(c, RuntimeCall::Balances(..)),
+            ProxyType::NonFungibile => !matches!(
+                c,
+                RuntimeCall::Balances(..)
+                    | RuntimeCall::SubtensorModule(pallet_subtensor::Call::add_stake { .. })
```

Comment:
Note to self: Rebase stao off development so we can capture this

Reply:
After rebase, the NonFungibile excludes all extrinsic related to balance in Subtensor pallet.

Reply:
ha.. this was a note for me !

Reply:
pls add stao extrinsic in filter